docker/README.md

```diff
@@ -46,6 +46,14 @@ and may make it harder to reproduce issues.
 run
 [docker-volume-watcher](https://github.com/merofeev/docker-windows-volume-watcher)
 
+#### DB tool:
+
+To connect to the newly created container db with psql/{db tool} you connect to it by default if not anything else specified via port 6543 on localhost with the `DATABASE_NAME` specificed as such:
+
+```bash
+$ psql postgres://$DATABASE_NAME@localhost:6543
```

Comment:
The database name goes into the path of the connection string, and you may need to add authentication credentials. These settings are from `.env`:

```suggestion
$ psql "postgres://$DATABASE_OWNER:$DATABASE_OWNER_PASSWORD@localhost:6543/$DATABASE_NAME"
```

Reply:
Cool! Do we need to say something about .env you think?

Reply:
Not sure